DevCycle.SDK.Server.Common/Model/Local/Variable.cs

```diff
@@ -88,14 +87,13 @@ public static Variable<T> InitializeFromVariable(Variable<T> variable, string ke
         [DataMember(Name = "value")]
         public T Value { get; set; }
 
-        [DataMember(Name = "defaultValue")]
-        public T DefaultValue { get; set; }
+        [DataMember(Name = "defaultValue")] public T DefaultValue { get; set; }
```

Comment:
This should probably be on the next line like all the other types.

Reply:
This was an auto-lint fix - not something I did myself